client/src/modules/locations/province/modal/createUpdate.js

```diff
@@ -0,0 +1,46 @@
+angular.module('bhima.controllers')
+  .controller('ProvinceAddController', ProvinceAddController);
```

Comment:
Is it possible to name the controller with something like `CreateEditProvinceController` or something else in this way.